tests/specs/ledger/ledger.spec.ts

```diff
@@ -17,6 +17,7 @@ test.describe('App with Ledger', () => {
   test('that you can navigate to activity page', async ({ homePage }) => {
     await homePage.clickActivityTab();
     const noActivityText = homePage.page.getByText('No activity yet');
-    await test.expect(noActivityText).toBeVisible();
+    // Account has activity to make sure we don't see label
+    await test.expect(noActivityText).not.toBeVisible();
```

Comment:
feel like this test will pass, even if there is error on activity page

Reply:
added a url check